src/sentry/identity/oauth2.py

```diff
@@ -261,6 +262,11 @@ def exchange_token(self, request, pipeline, code):
                 'error': 'Could not verify SSL certificate',
                 'error_description': u'Ensure that {} has a valid SSL certificate'.format(url)
             }
+        except JSONDecodeError:
+            return {
```

Comment:
Can we add some logging here, just so we're aware of how much this is happening

Reply:
👍 Done